src/Writers/HTMLWriter.jl

```diff
@@ -1030,7 +1030,11 @@ mdconvert(h::Markdown.Header{N}, parent; kwargs...) where {N} = DOM.Tag(Symbol("
 
 mdconvert(::Markdown.HorizontalRule, parent; kwargs...) = Tag(:hr)()
 
-mdconvert(i::Markdown.Image, parent; kwargs...) = Tag(:img)[:src => i.url, :alt => i.alt]
+mdconvert(i::Markdown.Image, parent; kwargs...) = if endswith(i.url, ".webm")
```

Comment:
Would be good to support other formats as well while we're at it I think. WebM (`video/webm`; `.webm`), MP4 (`video/mp4`; `.mp4`) and Ogg (`video/ogg`; `.ogg`, `.ogv`), going by the list [here](https://www.w3schools.com/tags/tag_video.asp)?